### ISSUE TITLE
Future of Streisand & alternatives?

### ISSUE THREAD

**Issue by nabilfreeman** [jlund/streisand-discussions#208]
Hi all, I've noticed that Streisand has recently been archived.

I presume this means the project has come to an end which is very sad news - did see it coming though, as many misc fixes and patches were required to keep the Ansible scripts working in recent years and there was little maintenance.

Internet security has surely moved on in the 6 years since Streisand was released.

Does anybody know where privacy-conscious users should look for browsing safely in the 2020s?

Thanks
Nabs

**Comment by nabilfreeman** [comment]
After some research I've come across Xray XLTS:

https://github.com/XTLS/Xray-core

Docker images are available, but slightly more of a learning curve for some Streisand users for sure.

**Comment by arrrgi** [comment]
If you're looking for something similar, do look at https://github.com/trailofbits/algo , it's a similar project and is still actively maintained.

**Comment by omidraha** [comment]
https://github.com/D1E7BF44/streisand

**Comment by StreamingKarl** [comment]
Another alternative is [Outline](https://getoutline.org/).
I installed both Algo and Outline and they work great. Outline is a bit simpler to setup though, especially when you use the DigitalOcean option.

**Comment by WoodpeckerBaby** [comment]
Note that Algo doesn't work well in censorship countries like China. Outline works in China, but you need to switch servers and reinstall everything all the time.

**Comment by romain130492** [comment]
> Note that Algo doesn't work well in censorship countries like China. Outline works in China, but you need to switch servers and reinstall everything all the time.

that's true, did you find an alternative to Outline that works well in China since? 
Since reinstalling a new server every 3 days is quite a burden

**Comment by nabilfreeman** [comment]
@romain130492 Vmess and even plain SS are both still very viable in China. I have a Streisand box still running with an uptime of nearly 5 years without an IP rotation.

**Comment by nickolasclarke** [comment]
@nabilfreeman an ol' streisand contributor here. I'm headed back to the mainland and trying to get my ducks in order. I'm interested to know where hosting and throughput your seeing these days.

**Comment by ignoramous** [comment]
> For folks looking for alternatives, Outline Foundation's project might be handy:
> - https://github.com/OutlineFoundation/outline-apps
>
> There's also SagerNet, Amnezia, and Hiddify from the anti-censorship community (these are cross platform and self-hostable):
>
> - https://github.com/amnezia-vpn/amnezia-client
> - https://github.com/SagerNet/sing-box
> - https://github.com/hiddify/Hiddify-Manager
>
> A like-for-like alternative is probably Trail's "algo":
> - https://github.com/trailofbits/algo

*From*: https://github.com/StreisandEffect/discussions/issues/209#issuecomment-4163845269
